Assets/Scripts/Health.cs

```diff
@@ -4,40 +4,23 @@
 
 public class Health : MonoBehaviour {
 
-	// Use this for initialization
 	public int maxHealth = 10;
+	public int currentHealth;
 
-    [SerializeField]
-    private int currentHealth;
+	public bool isDead { get { return (currentHealth <= 0); } }
 
-	private Vector2 initalScale;
-
-    [SerializeField]
-    private bool alive;
-   
-
-	public bool isDead { get { return !alive; } }
+	public int CurrentHealth { get { return currentHealth; } }
 
 	void Start () {
 		currentHealth = maxHealth;
-        alive = true;
-		initalScale = this.gameObject.transform.localScale;
 	}
 	
-	// Update is called once per frame
 	void Update () {
-		ResizeWithHealth ();
 		if (currentHealth <= 0) {
-			alive = false;
+			Destroy (this.gameObject);
```

Comment:
Ill allow this for this version because time nessasatates it, I will be putting down that full component design should be used in future.